PencilCase/PencilCase/www/css/custom.css

```diff
@@ -5,3 +5,12 @@
 .textbox-validation-error {
     border: 2px solid red;
 }
+
+.validation-error-message {
+    margin: -0.4em 0 0.5em;
+}
+
+.validation-error-message > span {
```

Comment:
The style for error message span element inside the container.
Since this is pure css but not Sass, so couldn't make it as nested content of the .validation-error-message, which is the way you are used to do in daily work.